content/post/2017-06-04-probability.Rmd

```diff
@@ -316,11 +327,11 @@ hist(z, breaks = 0:25)
 
 ```
 
-### Functions 
+### Functions
 
 [Bolker 2009 p 118](https://ms.mcmaster.ca/~bolker/emdbook/book.pdf)
 
-![](../../static/images/bestiary_functions.png)
+<!--![](../../static/images/bestiary_functions.png)-->
```

Comment:
couldn't find file `../../static/imagesbestiary_functions.png`, it wasn't in the directory.

I search the repo history for it, but couldn't find it